src/main/java/lk/ac/pdn/co328/studentSystem/RegistrationSystem.java

```diff
@@ -1,69 +1,69 @@
+/*
+E/12/302
+ */
 package lk.ac.pdn.co328.studentSystem;
 
+import java.util.ArrayList;
 import java.util.Scanner;
 
-public class RegistrationSystem
-{
-    static Scanner stdin = new Scanner(System.in);;
-    static StudentRegister register = new StudentRegister();
-    public static void main(String[] args)
-    {
+public class RegistrationSystem {
+    private static Scanner stdin = new Scanner(System.in);
+    private static StudentRegister register = new StudentRegister();
+
+    public static void main(String[] args) {
         System.out.println("Student management system command line version for CO328");
-        while(true)
-        {
-            try
-            {
+        while (true) {
+            try {
                 printSelection();
-                int command = Integer.parseInt(stdin.nextLine());
```

Comment:
No need for Integer.parseInt in here since we can scan a int using nextInt() so no need for this sugar
Also try catch blocks are really not needed in here I'm lazy so let them be in case of scanner class give an exception